src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -2,15 +2,16 @@
 /* eslint-disable class-methods-use-this */
 import { uniqBy } from 'lodash-es';
 import {
+  Contract,
   decode,
   encode,
   Encoded,
   Encoding,
-  getAddressFromPriv,
-  getHdWalletAccountFromSeed,
   isAddressValid,
+  MemoryAccount,
   Tag,
 } from '@aeternity/aepp-sdk';
+import { getHdWalletAccountFromSeed } from '@aeternity/aepp-sdk-13';
```

Comment:
I will make a new release of sdk to fix that. Meanwhile, you can use it as is since tree shaking will remove the unused code.